specification/app/resource-manager/Microsoft.App/preview/2025-02-02-preview/examples/ContainerApps_CreateOrUpdate.json

```diff
@@ -95,7 +95,14 @@
             "httpReadBufferSize": 30,
             "httpMaxRequestSize": 10,
             "logLevel": "debug",
-            "enableApiLogging": true
+            "enableApiLogging": true,
```

Comment:
recommend using the same example in CP unit test to catch issues with implementations